firefox_ui_harness/runners/base.py

```diff
@@ -7,8 +7,8 @@
 import mozinfo
 from marionette import BaseMarionetteTestRunner
 
-from firefox_ui_harness.testcases import FirefoxTestCase
 import firefox_ui_tests
+from firefox_puppeteer.testcases import FirefoxTestCase
```

Comment:
In a follow-up bug we will have to get rid of this import given that the harness should not reference puppeteer. For now this looks fine given that we at least have extracted the testcases from the harness.